themes/bootstrap3/js/combined-search.js

```diff
@@ -0,0 +1,22 @@
+/*global VuFind, setupOpenUrlLinks, checkSaveStatuses, setupSaveRecordLinks */
+VuFind.combinedSearch = (function() {
```

Comment:
Well, that's of course always up to debate, but we have used it extensively when adding our own JS to VuFind, and it has worked marvelously, so I'd say "yes". I think it was modeled after the module pattern described e.g. here: https://toddmotto.com/mastering-the-module-pattern/.